examples/src/IconsExample.js

```diff
@@ -42,7 +45,7 @@ class IconsExample extends React.Component<Props, State> {
                         const Component = icon.component;
                         return (
                             <div className="icon" key={icon.name}>
-                                <Component />
+                                {icon.content ? icon.content() : <Component />}
```

Comment:
To allow for docs that need multiple variations of a single component displayed, like this:

<img width="1089" alt="Screen Shot 2019-03-11 at 9 23 19 PM" src="https://user-images.githubusercontent.com/1280912/54174481-f70f8d80-4443-11e9-9230-152557a883a3.png">